app/services/event_driven_services.py

```
@@ -0,0 +1,766 @@
+                    )
+                    self.consumer_tasks[f"{service_name}_fallback"] = fallback_task
+                    
+                    logger.info(f"Started {service_name} service", priority=priority.value)
+                    
+                    # Stagger service starts to avoid resource spikes
+                    if priority == ServicePriority.CRITICAL:
+                        await asyncio.sleep(0.1)  # 100ms stagger for critical
+                    else:
+                        await asyncio.sleep(0.5)  # 500ms stagger for others
+                        
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)